gwcli/gateway.py

```diff
@@ -123,7 +123,8 @@ def export_ansible(self, config):
         ansible_vars.append("deploy_settings: true")
         ansible_vars.append("perform_system_checks: true")
         ansible_vars.append('gateway_iqn: "{}"'.format(config['gateways']['iqn']))
-        ansible_vars.append('gateway_ip_list: "{}"'.format(",".join(config['gateways']['ip_list'])))
+        ansible_vars.append('gateway_ip_list: "{}"'.format(
+            ",".join(config['gateways']['ip_list'])))
```

Comment:
I have been wondering what the coding style rule for this is, so I probably have been committing a lot of errors :)

Is the rule that if we are crossing 80 cols so we have to break up a line, but we can start on a new line and not have to break up that next line then start on a new line and indent over one chunk?

I was wondering we don't indent over so the new line starts under the "(" after "append(" similar to how the chunk below starts key=lambda under the "(" after "sorted("?

It seems fine. Just wondering.

Reply:
I have it set to warn at crossing 99 columns. This is just the standard PEP8/Flake8 Python style checker w/o anything special added (I did have to disable the complexity checker due to some of the huge ```def manage``` methods). After this merges, all PRs will get checked so that we have a consistent style (and catch silly errors), but you can also run locally by running ```tox -rv -e flake8```

Reply:
... and to answer your question about the difference, I think it's because this is within a second method call so dropping down to align w/ the ```(``` of ```append``` would imply the parameter is to that method when it's really to ```format```.